Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoAnalytics.kt

```diff
@@ -8,7 +8,14 @@ internal object VenmoAnalytics {
     const val TOKENIZE_SUCCEEDED = "venmo:tokenize:succeeded"
     const val APP_SWITCH_CANCELED = "venmo:tokenize:app-switch:canceled"
 
-    // Additional Detail Events
+    // Launching App Switch events
+    const val APP_SWITCH_STARTED = "venmo:tokenize:app-switch:started"
     const val APP_SWITCH_SUCCEEDED = "venmo:tokenize:app-switch:succeeded"
     const val APP_SWITCH_FAILED = "venmo:tokenize:app-switch:failed"
+
+    // Handle return events
+    const val HANDLE_RETURN_STARTED = "venmo:tokenize:handle-return:started"
+    const val HANDLE_RETURN_SUCCEEDED = "venmo:tokenize:handle-return:succeeded"
+    const val HANDLE_RETURN_FAILED = "venmo:tokenize:handle-return:failed"
+    const val HANDLE_RETURN_NO_RESULT = "venmo:tokenize:handle-return:no-result"
```

Comment:
Thoughts on adding this `no-result` event?

It would be an Android specific event, but it'd give us some insight into when buyers are canceling via the CCT X or entering CCT picture-in-picture mode.

Reply:
I think it would be helpful